infrastructure/tooling/src/smoketest/checks/role.js

```diff
@@ -0,0 +1,21 @@
+const taskcluster = require('taskcluster-client');
+
+exports.tasks = [];
+exports.tasks.push({
+  title: 'Create role and expand smoketest',
+  requires: [],
+  provides: [
+    'role-create-expand',
+  ],
+  run: async () => {
+    let role = new taskcluster.Auth(taskcluster.fromEnvVars());
```

Comment:
It's more typical to call this variable `auth`, since it's a link to the auth service.